src/async_fuse/memfs/s3_metadata.rs

```diff
@@ -54,6 +56,9 @@ const MY_TTL_SEC: u64 = 3600; // TODO: should be a long value, say 1 hour
 const MY_GENERATION: u64 = 1; // TODO: find a proper way to set generation
 /// S3 information string delimiter
 const S3_INFO_DELIMITER: char = ';';
+#[allow(dead_code)]
```

Comment:
Same above

Reply:
No, and the reason is same as above.

The constant is used only by `rename_exchange_helper`, over 7.23. Maybe I can move this constant into the method, but the constant may be used in other methods in the future, as it is a common constant.